prelude/README.md

```diff
@@ -111,6 +122,16 @@ instance categoryArr :: Category Prim.Function
 ($) :: forall a b. (a -> b) -> a -> b
 ```
 
+Applies a function to its argument
+
+```purescript
+    (+) 1 ((*) 2 (const 1 2)) = (+) 1 $ (*) 2 $ const 1 2
+```
+
+`($)` is different from [`(#)`](#-2) because it is right-infix instead of left. Right-infix
```

Comment:
I don't know if showing the parse would help a beginner, and they can get this info from the wiki anyway. I think the key is that you can rewrite the nested application `f (g (h x))` as `f $ g $ h x` or as `f <<< g <<< h $ x` or as `x # h >>> g >>> f`.